Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -1,36 +1,72 @@
 ﻿l_english: 
 
+
+ # NOTES ON IMPROVEMENTS TO THE CURRENT SET OF LOCALISATION IN CASE SOMEONE ELSE WANTS TO HAVE A REDRAFT AT A LATER POINT IN TIME (AS I AM NOT A WRITER):
+ #    - CHARACTER SPECIFIC EVENTS COULD BE REWRITTEN SOMEWHAT TO REFLECT MORE THE PERSONALITY OF SUCH CHARACTER BETTER:
+ #        - I.E. TEWDRIC/ARTHUR ARE HONOURABLE, STRATEGIC, ETC.; UTHER IS OLD, CANTANKEROUS AND CALLOUS; GUNDLEUS IS AMBITIOUS AND TWISTED, ETC.
+ #    - LOCALISATION COULD DO WITH APPROPRIATE SCOPING (I.E. TITLES, CHARACTER NAMES) WHERE THERE ISN'T ALREADY
+ #    - GRAMMAR MAY BE A BIT CRAP - DONE TOO MUCH OF THIS HYPENATING I THINK, AND MAY NOT BE APPROPRIATE EVERYWHERE. 
+ #    - A COUPLE OF DESCRIPTIONS ARE EMPTY AND UNFINISHED
+
  BTWK_story_event.0101.t:0 "The Winter King"
  BTWK_story_event.0101.desc:0 "Your one and only heir is your newborn grandson, Mordred, after your only legitimate son - his namesake - was killed in battle some months prior. The babe's crippled leg - a bad omen - only helps to sow the seeds of discontent. Without sufficient preparation for your death, and his ascension to rule, this surely will spell disaster for Dumnonia. \n\nAs High King of Britannia, you have the power to convene 'High Councils' in times of peril. The last of these was over 80 years ago, but with the threat ever-increasing encroachment by the Saxons, and British warlords forever infighting, you see no other option to bring peace to Britannia once more. Messengers have been sent you would like to meet them in Glevum."
  BTWK_story_event.0101.a:0 "I only hope it is not too late to heal old wounds."
 
  ############ PLACEHOLDER TEXT IN THIS BLOCK ############
  BTWK_story_event.0102.t:0 "The High Council at Glevum"
- BTWK_story_event.0102.desc:0 "Uther requests a convention at Glevum. Will you come? placeholder text"
- BTWK_story_event.0102.a:0 "I will honor the request of the High King."
- BTWK_story_event.0102.b:0 "He is no King of mine."
+ BTWK_story_event.0102.desc:0 "A messenger has delivered a letter bearing the insignia of Dumnonia. Opening it, you read that a High Council is to be held in the old Roman seat of Glevum. Being one of the foremost warlords of Britannia, your presence has been requested at the behest of High King Uther. While the meeting is of course not mandatory, the gathering certainly presents a great opportunity to heal old wounds between your fellow warlords. With the Germanic barbarians amassing their strength on the frontiers of LLoegyr, and the frequency of Irish raids increasing yearly, putting an end to the infighting plaguing Britons is of paramount importance. A missive within the letter particularly sticks out, however - Uther intends to find his newborn heir a foster father, fearing himself not long for this world. Perhaps this is an ideal opportunity to forge an alliance with the breadbasket of Britain herself - the Kingdom of Dumnonia."
```

Comment:
- Insignia -> seal (more period-appropriate)
- Could combined the second and third sentences, the current phrasing sounds a bit overly wordy (e.g. something like "Your presence has been requested by High King Uther Pendragon at a High Council to be held in the old Roman seat of Glevum.")
- Maybe "great hall of Glevum" is better, I don't think it was actually ever a Roman seat of anything important
- The Germanic barbarians aren't amassing their strength on the frontiers of Lloegyr... they've already overrun about a third of it
- Foster father -> guardian, I don't think it was intended for Mordred to actually stay with them

Reply:
1. Noted.
2. Noted. 
3. It was labelled Roman per its description within TWK as "... a Roman town ..." (start of Part 1 Chapter 3, pg. 45). I will change seat to town to avoid confusion. Given Glevum is supposed to be Gloucester I would say the usage is fine, unless you think otherwise.
4. Does "amassing their strength within the Lost Lands of LLoegyr" sound more appropriate to you? 
5. Think my usage of "foster father" has been a big case of Mandela Effect (unless it was so within the CK2 version?) as I can't find its usage anywhere within the book. Will a description of it as "the Guardian through Marriage" (or the like) be okay? I'll change the loc elsewhere as appropriate.

Reply:
3. Town is fine, yes (could even use city TBH).
4. Yeah that's good.
5. Probably just confusing it with the however other many characters actually had a foster father. 😅 Do you recall how they described it in the books? I'm using "guardian" because that's what I recall but I haven't actually checked...

Reply:
3. Haven't checked for errors, but this is the current redraft I have: 
"A messenger has delivered a letter bearing the seal of Dumnonia. King Uther has arranged for a High Council to be convened in the old Roman town of Glevum. As you are one of Britain's most powerful warlords, he has requested your attendance. Presence at the meeting offers a great opportunity to heal the old wounds between rival realms, giving chances to forge bonds anew. With the Germanic barbarians amassing their strength within the Lost Lands of LLoegyr, and the frequency of Irish raids increasing yearly, putting an end to the infighting plaguing Britons is of paramount importance. A missive within the letter particularly sticks out however - Uther intends to seek marriage for his late heir's widow, Princess Norwenna. Any unmarried ruler, or his unmarried heir, is an eligible candidate for this arrangement, and will be oathbound in the protection of the Edling Mordred - the new heir presumptive of Dumnonia - till he is of fit age to rule. To fulfil such duties, an alliance will be formed between the chosen realm, and Dumnonia herself."

Haven't checked for typos yet.

Reply:
- Only the first L in Lloegyr should be capitalized
- "...till he is of fit age to rule. To fulfil such duties, an alliance will be formed between the chosen realm, and Dumnonia herself" -> "...till he is of fit age to rule, effectively creating an alliance with Dumnonia." would be shorter and a little awkward?

Otherwise looks good I think.

Reply:
"...till he is of fit age to rule, effectively creating an alliance with Dumnonia." Agree with this change, and has been implemented.

5. 'Do you recall how they described it in the books? I'm using "guardian" because that's what I recall but I haven't actually checked...', I think guardian/protector is more appropriate really. Scanned through the council text again and it names Gundleus as a protector of mordred only (essentially regent presiding over a council, but an alliance will have to suffice), surrounded by a council comprised of Dumnonians and Gwentians. Foster father isn't the right term imo. As for the other guardians, they were death-oath takers loyal from Dumnonia, and also Tewdric (as a friend of Dumnonia).